orion/pkg/domains/regex.go

```diff
@@ -22,4 +22,19 @@ const (
 
 	// Ensures at least one uppercase or lowercase letter
 	REGEX_LETTER = `[A-Za-z]+`
+
+	// Ensures formatting matches a valid street address
+	REGEX_STREET = `^[1-9][0-9]*( [A-Z][a-z]*){2,}$`
+
+	// One or more capitalized words
+	REGEX_CITY = `^([A-Z][a-z]*)( [A-Z][a-z]*)*$`
+
+	// Two letter state codes
+	REGEX_STATE = `^[A-Z][A-Z]$`
+
+	// Ensures formatting matches a valid zipcode
+	REGEX_ZIPCODE = `^[0-9]{5}(-[0-9]{4}){0,1}$`
+
+	// Ensures at least one alphanumeric character
+	REGEX_ALPHA = `[[:alnum:]]`
```

Comment:
nice! Can you put this up as the first REGEX value? The idea is keep the more generic values up top so they're easy to find and the more detailed / granular regex at the bottom

Reply:
In the mean time could you also move REGEX_AT_LEAST_ONE_LETTER up top as second right below REGEX_ALPHA?